src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -1836,6 +1836,58 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
 #End Region
 
+#Region "Operations"
+
+        ''' <summary>
+        ''' Gets the low-level operation corresponding to the method's body.
+        ''' </summary>
+        ''' <param name="method">The method symbol.</param>
+        ''' <param name="cancellationToken">An optional cancellation token.</param>
+        ''' <returns>The low-level operation corresponding to the method's body.</returns>
+        Protected Overrides Function GetOperationCore(method As IMethodSymbol, Optional cancellationToken As CancellationToken = Nothing) As IOperation
+            Dim result As IOperation = Nothing
+            Dim vbmethod = method.EnsureVbSymbolOrNothing(Of MethodSymbol)(NameOf(method))
+
+            If vbmethod IsNot Nothing Then
```

Comment:
>If vbmethod IsNot Nothing Then [](start = 12, length = 30)

We also need to make sure the method is actually part of the source module of this compilation.